.github/workflows/translations.yml

```diff
@@ -0,0 +1,55 @@
+#  SuperTux
+#  Copyright (C) 2024 Tobias Markus
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 3
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+name: translations
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 0 * * 0'
+
+jobs:
+  update-all-translations:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout SuperTux repository
+        uses: actions/checkout@v2
+        with:
+          repository: 'SuperTux/supertux'
+          fetch-depth: 1
+
+      - name: Update translations
+        env:
+          TX_TOKEN: ${{ secrets.TRANSIFEX_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Installs transifex client:
+          curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
+          mv tx /usr/local/bin/tx
+
+          # Pull repository:
+          git fetch origin
+          git checkout -b master -t origin/master
+
+          # Pull translations:
+          tx pull --use-git-timestamps -a -f
+
+          # Commit
+          git status
+          git config --global user.email "SuperTux-bot@users.noreply.github.com"
```

Comment:
This is probably not the correct email, as `@users.noreply.github.com` emails consist of a few more numbers in the beginning. Example:

```
78196474+Vankata453@users.noreply.github.com
```

Reply:
so you're gonna invent a user id for an email? Also, it seems to work

Reply:
> Also, it seems to work

Have there been test pushes done with this email?

Reply:
Per [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address):

> If you created your account on GitHub.com prior to July 18, 2017, and enabled Keep my email address private prior to that date, your noreply email address from GitHub is [USERNAME@users.noreply.github.com](mailto:USERNAME@users.noreply.github.com).

It *might* be fine, but I don't know the bot account's settings.

Reply:
Considering that e-mail is now present, I'm gonna close this discussion